lib/lost-column.js

```diff
@@ -184,18 +184,11 @@ module.exports = function lostColumnDecl(css, settings) {
 
       newBlock(
         decl,
-        ':nth-child(1n)',
-        ['float', 'clear', 'margin-right', 'width'],
-        ['none', 'none', 0, 'auto']
-      );
-
-      newBlock(
-        decl,
         ':nth-child(1n + 1)',
         ['float', 'clear', 'margin-right', 'width'],
         ['none', 'none', 0, 'auto']
       );
-
```

Comment:
The only difference between #290 and this PR is this line.

Would love to not have this tab included.